kytos/core/controller.py

```diff
@@ -157,9 +157,38 @@ def __init__(self, options=None, loop=None):
 
     def enable_logs(self):
         """Register kytos log and enable the logs."""
+        decorators = self.options.logger_decorators
+        decorators = [self._resolve(decorator) for decorator in decorators]
+        LogManager.decorate_logger_class(*decorators)
         LogManager.load_config_file(self.options.logging, self.options.debug)
         LogManager.enable_websocket(self.api_server.server)
         self.log = logging.getLogger(__name__)
+        self._reload_core()
+
+    @staticmethod
+    def _resolve(name):
```

Comment:
@Ktmi, this `_resolve` ended up really handy, however for maintainability, I think it could be simplified, also the std lib docs recommends that direct use of [__import__()](https://docs.python.org/3/library/functions.html#import__) is also discouraged in favor of [importlib.import_module()](https://docs.python.org/3/library/importlib.html#importlib.import_module). Could you refactor this to use `importlib.import_module`? Something like this patch or similar if you manage to find a simpler solution:

```
diff --git a/kytos/core/controller.py b/kytos/core/controller.py
index e34cae1..cb05e93 100644
--- a/kytos/core/controller.py
+++ b/kytos/core/controller.py
@@ -13,6 +13,7 @@ Basic usage:
     controller = Controller(config.options)
     controller.start()
 """
+import importlib
 import asyncio
 import atexit
 import json
@@ -168,17 +169,10 @@ class Controller:
     @staticmethod
     def _resolve(name):
         """Resolve a dotted name to a global object."""
-        name = name.split('.')
-        used = name.pop(0)
-        found = __import__(used)
-        for name_part in name:
-            used = used + '.' + name_part
-            try:
-                found = getattr(found, name_part)
-            except AttributeError:
-                __import__(used)
-                found = getattr(found, name_part)
-        return found
+        name_split = name.split(".")
+        mod, attr = ".".join(name_split[:-1]), name_split[-1]
+        mod = importlib.import_module(mod)
+        return getattr(mod, attr)

     @staticmethod
     def _reload_core():
```

Reply:
Now using `importlib` for importing.